slices/util-linux.yaml

```
@@ -0,0 +1,271 @@
+      /etc/pam.d/su-l:
+
+  services:
+    essential:
+      - util-linux_bins
+    contents:
+      /usr/lib/systemd/system/fstrim.service:
+      /usr/lib/systemd/system/fstrim.timer:
+
+  copyright:
+    contents:
+      /usr/share/doc/util-linux/copyright:
+
+  blk-dev:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libmount1_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libudev1_libs
+    contents:
+      /usr/bin/addpart:
+      /usr/bin/delpart:
+      /usr/bin/lsblk:
+      /usr/bin/partx:
+      /usr/bin/resizepart:
+      /usr/sbin/blkdiscard:
+      /usr/sbin/blkid:
+      /usr/sbin/blkzone:
+      /usr/sbin/blockdev:
+      /usr/sbin/zramctl:
+
+  file:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libmount1_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/fallocate:
+      /usr/bin/flock:
+      /usr/bin/hardlink:
+      /usr/bin/lslocks:
+      /usr/bin/more:
+      /usr/bin/rename.ul:
+
+  file-system:
+    essential:
+      - libblkid1_libs
+      - libc6_libs
+      - libcap-ng0_libs
+      - libmount1_libs
+      - libpcre2-8-0_libs
+      - libselinux1_libs
+      - libsmartcols1_libs
+      - libudev1_libs
+      - libuuid1_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/findmnt:
+      /usr/bin/mountpoint:
+      /usr/sbin/findfs:
+      /usr/sbin/fsck:
+      /usr/sbin/fsck.cramfs:
+      /usr/sbin/fsck.minix:
+      /usr/sbin/fsfreeze:
+      /usr/sbin/fstrim:
+      /usr/sbin/isosize:
+      /usr/sbin/mkfs:
+      /usr/sbin/mkfs.bfs:
+      /usr/sbin/mkfs.cramfs:
+      /usr/sbin/mkfs.minix:
+      /usr/sbin/mkswap:
+      /usr/sbin/swaplabel:
+      /usr/sbin/switch_root:
+      /usr/sbin/wipefs:
+
+  # The following emulates the post-install script.
+  generated:
+    essential:
+      # Since the more binary is located at ./usr/bin/more and /bin in noble
+      # points to /usr/bin via base-files, the base-files_bin slice is included
+      # here.
+      - base-files_bin
+      - util-linux_file
+    contents:
+      /usr/bin/pager: {symlink: /bin/more}
+
+  setarch:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/bin/i386: { arch: [i386, amd64] }
+      /usr/bin/linux32:
+      /usr/bin/linux64:
+      /usr/bin/ppc: { arch: [ppc64el] }
+      /usr/bin/ppc32: { arch: [ppc64el] }
+      /usr/bin/ppc64: { arch: [ppc64el] }
+      /usr/bin/s390: { arch: [s390x] }
+      /usr/bin/s390x: { arch: [s390x] }
+      /usr/bin/setarch:
+      /usr/bin/x86_64: { arch: [amd64] }
+
+  proc:
+    essential:
+      - libc6_libs
+      - libcap-ng0_libs
+      - libsmartcols1_libs
+    contents:
+      /usr/bin/choom:
+      /usr/bin/chrt:
+      /usr/bin/ionice:
+      /usr/bin/prlimit:
+      /usr/bin/setpriv:
+      /usr/bin/setsid:
+      /usr/bin/taskset:
+      /usr/bin/uclampset:
+
+  text-dev:
```

```suggestion
  tty:
```

Or, maybe `terminal`?

---

Renamed as `tty` in 158ccf2.